hack/env

```diff
@@ -31,12 +31,12 @@
 #
 
 # NOTE:   only committed code is built.
-source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+OS_TMP_ENV_SET=y source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
```

Comment:
smells bad

Reply:
This is to prevent os::util::environment::setup_tmpdir_vars from being called, otherwise we are making a quite a few unnecessary temporary directories that are not used by hack/env.

Reply:
Are they not? The output from `os::log::*` for instance will go to `_output/scripts/env/logs/script.log`

Reply:
I've removed it with the latest push.